src/components/Tool/TntConnect/TntConnect.tsx

```diff
@@ -174,10 +178,6 @@ const TntConnect: React.FC<Props> = ({ accountListId }: Props) => {
       return;
     }
 
-    if (tntFile) {
```

Comment:
The old blob will already be released by the `useEffect` cleanup function on line 164.